apps/balancer-tools/src/app/apr/(utils)/getBalEmission.ts

```diff
@@ -0,0 +1,28 @@
+interface BalEmissions {
+  year: number;
+  weekly: number;
+  total_supply: number;
+}
+
+export const getBalEmissions = (year: number): BalEmissions => {
```

Comment:
this could come from [BalancerSDK](https://github.com/balancer/balancer-sdk/blob/379883359a0dead100282158f7aa2d13c255a7a3/balancer-js/src/modules/data/bal/emissions.ts)

You can see how the defyltica folks implemented it [here](https://github.com/Xeonus/balancer-analytics/blob/8722b3776340a764494b11c16cdb4c6ef3bcdda0/src/pages/Emissions/index.tsx#L68)

Reply:
nvm -- that'd make us need to install ethers again 🤦 let's just keep as is

Reply:
We could yoink just [emissions.ts](https://github.com/balancer/balancer-sdk/blob/379883359a0dead100282158f7aa2d13c255a7a3/balancer-js/src/modules/data/bal/emissions.ts)

Reply:
yea that's what I did here https://github.com/bleu-studio/balancer-tools/pull/205/commits/7947adf430cf49aaab1b060ec985dc20efd86217 😅

Reply:
cool implementation nevertheless -- but missing docs for how you came up with all that. See their excellent docs in their own implementation for reference